src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx

```diff
@@ -172,6 +167,34 @@ export default class ColumnCustomizationContent extends Vue {
     }
   }
 
+  _handleSelectColumn() {
+    const columnsSelected: string[] = [];
+    const filterSelectedColumns: DataTableColumn[] = [];
+
+    if (this._selectedStandardColumns) {
+      Array.from((this._selectedColumnsComponent?.$refs.select as HTMLSelectElement).selectedOptions).forEach(
+        (option: HTMLOptionElement) => {
+          columnsSelected.push(option.value);
+        }
+      );
+
+      columnsSelected.forEach((columnSelected: string) => {
+        this._selectedStandardColumns?.forEach((standardColumn: DataTableColumn) => {
+          if (columnSelected === standardColumn.name) {
+            filterSelectedColumns.push(standardColumn);
+          }
+        });
+      });
+    }
+
+    this._isSelectedColumnLocked = filterSelectedColumns.some((column: DataTableColumn) => column.locked);
+    this.$forceUpdate();
```

Comment:
I don't recommend using this. Can you re-think your solution and implement it in a way to not use forceupdate?

Reply:
I agree with @aryaroudi , forceUpdate should be avoided. @josecarlosgonzalezv

Reply:
Yes, I am agree too after reading some docs about it. 

After trying some different things, I decided to use a ref to manipulate the disabled attribute, you can check it now.